applications/views.py

```diff
@@ -98,9 +99,14 @@ def applications_csv(request, city):
     response = HttpResponse(content_type='text/csv')
     response['Content-Disposition'] = u'attachment; filename="{}.csv"'.format(city)
     writer = csv.writer(response)
-    writer.writerow(["Application Number", "email", "Application State", "RSVP Status", "Average Score"])
+    csv_header = ["Application Number","Application State", "RSVP Status", "Average Score"]
+    # replace commas with semicolons so it won't confuse csv parsers
+    csv_header.extend([striptags(question).replace(',', ';') for question in Question.objects.values_list('title', flat=True)])
```

Comment:
Great - it seems to be loading the correct set of questions now :confetti_ball:

Re. the CSV - the csv module will surround the value with quotation marks so that csv readers will interpret it correctly. It shouldn't need anything else.

Are you seeing this cause problems with any particular CSV reader?